migrations/V10__init_schema.sql

```diff
@@ -49,5 +49,6 @@ CREATE TABLE transaction
     -- make use of the transaction index in its own SQL queries.
     index JSONB NOT NULL
 );
+ALTER TABLE transaction ADD CONSTRAINT transaction_pk PRIMARY KEY (block_height, index);
```

Comment:
Changed this to have an explicit primary key to make upserts more intuitive, and prevent duplicate transactions from the same block being inserted more than once